src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -325,6 +325,242 @@ private AdsDataProvider() {
   }
 
   @DataProvider
+  public static Object[][] dfpParamsOasis() {
+    return new Object[][]{
+        {
+            "yugioh",
+            "Dark_Magician",
+            "wka.ent/_yugioh//article",
+            "TOP_LEADERBOARD",
+            Arrays.asList(
+                "\"s0\":\"ent\"",
+                "\"s0v\":\"comics\"",
+                "\"s0c\":[\"gaming\",\"anime\",\"videogames\"]",
+                "\"s1\":\"_yugioh\"",
+                "\"s2\":\"article\"",
+                "\"dmn\":\"wikiacom\"",
+                "\"hostpre\":\"yugioh\"",
+                "\"ar\":\"4:3\"",
+                "\"artid\":\"278\"",
+                "\"wpage\":\"dark_magician\"",
+                "\"ref\":\"direct\"",
+                "\"lang\":\"en\"",
+                "\"pv\":\"1\"",
+                "\"top\":\"1k\"",
+                "\"esrb\":[\"everyone\"]",
+                "\"age\":[\"teen\",\"kid\",\"18-34\"]",
+                "\"gnre\":[\"anime\",\"fantasy\",\"strategy\",\"comics\"]",
+                "\"media\":[\"tv\",\"cards\"]",
+                "\"pform\":[\"pc\",\"psp\"]",
+                "\"sex\":[\"m\"]"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\"sex\":[\"m\"]" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)